features/bootstrap/SecurityContext.php

```diff
@@ -22,7 +24,19 @@ public function iAmLoggedAs(string $email): void
             throw new \Exception(sprintf('Adherent %s not found', $email));
         }
 
-        $this->logAs($user, 'admin');
+        $this->logAs($user);
+    }
+
+    /**
+     * @When I am logged as :email admin
+     */
+    public function iAmLoggedAsAdmin(string $email): void
+    {
+        if (!$user = $this->getAdministratorRepository()->loadUserByUsername($email)) {
+            throw new \Exception(sprintf('Admin %s not found', $email));
```

Comment:
What about using `webmozart/assert` in beta contexts? It's easier to throw exceptions wit it.

Reply:
Why not But I would prefer to keep that for later.